mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -184,16 +186,40 @@ def handle_fallback(self, message):
 
             self.speak(response)
         else:
-            self.speak("Sorry, I don't understand your request.")
+            if len(others) == 1:
+                self.speak("Sorry, I didn't understand " +
+                           utterance + ". Searching for " +
+                           others[0] + " instead")
```

Comment:
In this case, Mycroft understood the utterance, but could not find the valid result. I think we should improve this feedback by changing `Sorry, I didn't understand` to something like: `Sorry, I didn't find an answer to utterance. I'm searching for others[0] instead`. In addition, these should be in a .dialog instead of hard-coded (see also comment below).